.github/workflows/additional.yml

```diff
@@ -0,0 +1,218 @@
+name: Additional
```

Comment:
This file looks very similar to the existing `tests.yaml` file. Perhaps it would be straightforward to just add a new build matrix entry for `mindeps` over in `tests.yaml`. What do you think?

Reply:
That sounds reasonable to me! So far think the major differences here would be:

- skipping over `host_info.py` since the benchmark there requires numpy
- maybe handling caching a little differently since we theoretically shouldn't need to update the mindeps cache very often

Reply:
> skipping over host_info.py since the benchmark there requires numpy

Ah, good point. Adding a (psuedocode) `if: not mindeps` to that step seems nice and lightweight 

> maybe handling caching a little differently since we theoretically shouldn't need to update the mindeps cache very often

Sorry, I don't quite follow here. Why would environment caching need to differ for the mindeps build vs. "normal" test runs?

Reply:
> Sorry, I don't quite follow here. Why would environment caching need to differ for the mindeps build vs. "normal" test runs?

Right now, we're constructing the conda environment cache key using the current date, so that regardless of changes to the conda environment files, the environment is reconstructed at least once a day:

https://github.com/dask/distributed/blob/27a91dd545d3638b6c3a6681089f0ceb91ed6b7d/.github/workflows/tests.yaml#L89-L104

For the test runs with unpinned dependencies, this is important because we want to regularly pull in the latest versions, but for the mindeps runs this isn't as much of an issue since the majority of the packages we're pulling in are years old are unlikely to be receiving new builds (outside of the `pytest-*` packages, which are unpinned). Thus, we could potentially change the logic for these runs so that the cached environment is kept for longer, which would speed things up a bit.

I haven't made the change yet as I wanted to make sure we aren't risking unexpected failures by caching the `pytest-*` packages, but if you think this is a reasonable change happy to do that as well 🙂